WebContent/WEB-INF/jsp/statsRdcpDownload.jsp

```diff
@@ -2,6 +2,9 @@
 <%@ page errorPage="/jsp/errorPage.jsp"%>
 <%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
 <%@ taglib uri="http://java.sun.com/jsp/jstl/fmt" prefix="fmt" %>
+<c:set var="viewTitle">${model.unique == "unique" ? " (Unique Views)" : " (Views)"}</c:set>
```

Comment:
Same question re: `unique` as above. (and apologies if i missed it in looking through the diff)

Reply:
See my comments above.